src/vscode-bicep/src/tree/TreeManager.ts

```diff
@@ -9,42 +9,27 @@ import { AzManagementGroupTreeItem } from "./AzManagementGroupTreeItem";
 import { AzResourceGroupTreeItem } from "./AzResourceGroupTreeItem";
 
 export class TreeManager extends Disposable {
-  private _azLocationTree: AzExtTreeDataProvider;
-  private _azManagementGroupTreeItem: AzExtTreeDataProvider;
-  private _azResourceGroupTreeItem: AzExtTreeDataProvider;
-
   constructor(private readonly outputChannelManager: OutputChannelManager) {
```

Comment:
Changes here were made to ensure newly created items in azure portal show up when deploy command is invoked.